content/en/blog/2021/better-external-authz/index.md

```diff
@@ -543,3 +543,5 @@ authorization system with the following benefits:
 
 We're working to promote this feature to a more stable stage in following versions and welcome your feedback at
 [discuss.istio.io](https://discuss.istio.io/c/security/).
+
+Thanks to `Craig Box`, `Christian Posta` and `Limin Wang` for reviewing drafts of this blog.
```

Comment:
```suggestion
_Thanks to Craig Box, Christian Posta and Limin Wang for reviewing drafts of this blog._
```

(hopefully that doesn't break spelling checker)

Reply:
I tried but it sill breaks the spelling checker, so reverted to use `.